sandbox/gas/tube/go

```diff
@@ -64,17 +65,46 @@ class DiaphragmIAnchor(sc.MeshAnchor):
         svr.soln[:,2].fill(0.0)
         if svr.ndim == 3:
             svr.soln[:,3].fill(0.0)
-        svr.soln[:,svr.ndim+1].fill(self.p1/(gamma-1))
+        # pressure
+        svr.soln[:,svr.ndim+1].fill(self.p1)
         # set.
-        slct_int = svr.blk.clcnd[:,2] > 0.5
-        slct_gst = svr.blk.gstclcnd[:,2] > 0.5
+        tube_middle = self.tube_height / 2
+        slct_int = svr.blk.clcnd[:,2] > tube_middle
+        slct_gst = svr.blk.gstclcnd[:,2] > tube_middle
         slct = np.concatenate((slct_gst, slct_int))
         svr.soln[slct,0] = self.rho2
-        svr.soln[slct,svr.ndim+1] = self.p2/(gamma-1)
+        svr.soln[slct,svr.ndim+1] = self.p2
```

Comment:
Thanks for the comment. I think the data array is ``[rho, vx, vy, vz, p]`` for each cell, that is, I think it is pressure in 3D cases.

I refer to [line 48 of this legacy driving script](https://github.com/solvcon/solvcon/blob/master/examples/gasdyn/tube/go#L48) to implement this line. Thus, comparing [line 48](https://github.com/solvcon/solvcon/blob/master/examples/gasdyn/tube/go#L48) to [line 44 of the same legacy driving script]( https://github.com/solvcon/solvcon/blob/master/examples/gasdyn/tube/go#L44) , [line 48](https://github.com/solvcon/solvcon/blob/master/examples/gasdyn/tube/go#L48), which is ``svr.soln[slct,svr.ndim+1] = self.p2`` as well, is a typo, isn't it?